### PR TITLE
Updated the dependency check based on AutoPilot

### PR DESCRIPTION
**Pull request by avinash-palleti**
Signed-off-by: Avinash Reddy Palleti <avinash.reddy.palleti@intel.com>

### CODE REVIEW

File: testbed/testbed.sh [hunk 1]
```diff
@@ -58,7 +58,11 @@ test_dep () {
 check_deps () {
     test_dep gz
     test_dep socat
-    test_dep $APM_CMD
+    if (("$AUTOPILOT" == "$AP_PX4")); then
+	test_dep $PX4_CMD
+    else
+        test_dep $APM_CMD
```
Comment (rchiossi) [inline testbed/testbed.sh]: Wrong indentation.

Comment (avinash-palleti) [inline testbed/testbed.sh]: Thanks Rodrigo. Will update this indentation issue in my next patch.

**Comment by rchiossi** [comment]
Apparently your editor is mixing spaces and tabs, as the second patch just made things worse. Also, we usually squash error fix commits like that second one and just push the functional ones. For big patches, it is better open a new PR with a V2 tag, but for small ones like this you can just force push a new version.

As this was fairly simple, I fixed the indentation and pushed it to master. :)

**Comment by avinash-palleti** [comment]
Thanks Rodrigo, I will check out the issue with my editor.

**Pull request closed without merging**